src/components/Person/Person.jsx

```diff
@@ -2,0 +1,19 @@
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">
+      {person.age ? <p>I am ${person.age}</p> : null}
+    </p>
+    {person.isMarried === true ? (
+      <p className="Person__partner">
+        {person.sex === 'm' ? (
+          <p>${person.partnerName} is my wife</p>
```

Comment:
The template literal syntax is incorrect here. It should be `{`${person.partnerName} is my wife`}` instead of `<p>${person.partnerName} is my wife</p>`.